examples/sequence-to-sequence/seq2seq_translator/seq2seq_pytorch.py

```diff
@@ -0,0 +1,378 @@
+# Requirements
+
+from __future__ import unicode_literals, print_function, division
+import io
+import unicodedata
+import re
+import random
+import time
+import math
+import torch
+import torch.nn as nn
+from torch import optim
+import torch.nn.functional as F
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+r = random.SystemRandom()
+
+# Data Preparation
+
+SOS_token = 0
+EOS_token = 1
+
+
+class Lang:
```

Comment:
Can you put this class and all the other utilities that are not dependent on the framework in a separate file like `utils.py`? To avoid code duplication